css/style.css

```
@@ -0,0 +1,63 @@
+/*!
+ * All Rights Reserved
+ * This software is proprietary information of
+ * AKUREY
+ * Use is subject to license terms.
+ * Filename: style.css
+ */
+
+h1 {
+    width: 100%;
+    text-align: center;
+}
+
+a  {
+    color: rgb(253, 253, 253);
+    text-decoration: none;
+}
+
+a:hover  {
+    color: rgb(95, 233, 157);
+}
+
+p {
+    color: rgb(253, 253, 253);
+    font-size: 130%;
+}
+
+.containerbox {
```

As naming convention, use hyphen (-) to separate words
```suggestion
.container-box {
```

More information: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/ 
Later, you will learn about BEM